api/openapi/my_api.yaml

```diff
@@ -468,6 +468,10 @@ components:
           type: string
           description: "Provides the client with grouping patient variables into multiple groups"
           example: "Body Type"
+        certitude:
+          type: integer
+          description: The certitude of the value, 0 not certain, 1 somewhat certain, 2 certain
```

Comment:
Should we quota the description value as a string similarly as other description fields?

Reply:
But certitude has an integer value 0, 1 or 2. Look at how certitude is defined here: https://github.com/RENCI/pds/blob/master/api/openapi/my_api.yaml at line 521.

Reply:
OK. Then I guess open api spec does not require to put description string within the quotation marks.